src/runtime_src/core/pcie/emulation/common_em/CMakeLists.txt

```diff
@@ -52,3 +52,9 @@ target_link_libraries(common_em
   rt
   )
 
+install(TARGETS common_em
+  EXPORT xrt-dev-targets
+  ARCHIVE DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT}
+  LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT} ${XRT_NAMELINK_ONLY}
+)
```

Comment:
I would suggest to include common_em_objects in the static shim emulation libraries and not release common_em library.